docs/deployments/gh_pages.mdx

```diff
@@ -0,0 +1,16 @@
+---
+id: gh_pages
+slug: /gh_pages
+title: GitHub Pages
+description: Guide for deploying a task to GitHub Pages
+---
+
+The `.github/workflows/release.yml` workflow will automatically deploy your task to GitHub pages if you have it enabled for your repository. It uploads the built website to the `gh-branch` branch. You can then access your task at `<github user>.github.io/<repository>`.

```

Comment:
Is `gh-branch` supposed to be `gh-pages`? (You refer to it as `gh-pages` in a later paragraph, and also that's the title of the file.)

```suggestion
The `.github/workflows/release.yml` workflow will automatically deploy your task to GitHub pages if you have it enabled for your repository. It uploads the built website to the `gh-pages` branch. You can then access your task at `<github user>.github.io/<repository>`.

```

Reply:
Yup! Good eye!